com.unity.sg2/Editor/GraphDeltaRegistry/FunctionDefinitions/StandardDefinitions/Utility/HDRP/EmissionNode.cs

```diff
@@ -100,6 +100,11 @@ internal class EmissionNode : IStandardNode
                     displayName: "Exposure Weight",
                     tooltip: "Controls how much the exposure affects the emission."
                 ),
+                new ParameterUIDescriptor(
+                    name: "NormalizeColor",
+                    displayName: "Normalize Color",
+                    tooltip: "Ensures the channels of the color are between zero and one."
+                ),
```

Comment:
The NormalizeColor static bool didn't show up so I added this ParameterUIDescriptor for it and now it does appear.